tests/drivers/clock_control/stm32_clock_configuration/stm32u5_core/boards/pll_hse_fracn_160.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Warning: This overlay performs configuration from clean sheet.
+ * It is assumed that it is applied after clear_clocks.overlay file.
+ */
+
+/*
+ * Warning: HSE is not implemented on available boards, hence:
+ *          This configuration is only available for build
+ */
+
+&clk_hse {
+	status = "okay";
+	clock-frequency = <16777216>;
+};
+
+&pll1 {
+	div-m = <2>;
+	mul-n = <19>;
+	div-p = <1>;
+	div-q = <1>;
+	div-r = <1>;
+	fracn = <602>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(160)>;
```

Comment:
This is the freq I'd expect to be modified

Reply:
I am not quite sure about this. From my understanding, the RCC will use PLL as the clock source with a 160 MHz value. The 160 MHz value is the PLL output that was generated using the 16777216 Hz HSE. 

Please let me know if I am missing something.

Reply:
Ok, I was the one confused. Fine for me, thanks